src/reference/asciidoc/kafka.adoc

```diff
@@ -2732,6 +2785,57 @@ The timers are named `spring.kafka.template` and have the following tags:
 
 You can add additional tags using the template's `micrometerTags` property.
 
+[[micrometer-native]]
+===== Micrometer Native Metrics
+
+Starting with version 2.5, the framework provides <<factory-listeners>> to manage Micrometer `KafkaClientMetric` s whenever producers and consumers are created and closed.
```

Comment:
Probably you really mean it as a plain text "Kafka client metrics".
Looks like there is no `KafkaClientMetric` type...

Reply:
It's `KafkaClientMetrics`.